core-web/libs/sdk/react/src/lib/components/Container/Container.tsx

```diff
@@ -42,6 +40,8 @@ export interface ContainerProps {
 }
 
 export function Container({ containerRef }: ContainerProps) {
+    const { isInsideEditor } = useContext(PageContext) as PageProviderContext;
```

Comment:
The cast is necessary because PageContext can be null
I can merge both like
`const { isInsideEditor } = useContext<PageProviderContext>(PageContext) as PageProviderContext;`

But i think `<PageProviderContext>` is redundant in this case

Reply:
Why can it be `null`, @KevinDavilaDotCMS ?

Reply:
Might be because it get's init in `null`?